src/core/mount-setup.c

```diff
@@ -478,7 +478,7 @@ static int relabel_extra(void) {
 }
 #endif
 
-int mount_setup(bool loaded_policy) {
+int mount_setup(bool loaded_policy, bool skip_setup) {
```

Comment:
please name this argument by what it does, i.e. "leave_propagation" or so.

Reply:
Thanks a lot, we will fix it as soon as possible.